### PR TITLE
[system test] - fix removal of namespace after test run

### PR DESCRIPTION
**Pull request by see-quick**
### Type of change

- Bugfix

### Description

This PR adds a check for removing the cluster operator namespace at the end of the test execution.

### Checklist

- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java [hunk 1]
```diff
@@ -381,13 +382,26 @@ public void updateSubscription(OlmConfiguration olmConfiguration) {
...
+                cluster.createNamespaces(CollectorElement.createCollectorElement(testClassName, testMethodName), namespaceInstallTo, bindingsNamespaces);
+                StUtils.copyImagePullSecrets(namespaceInstallTo);
+
+                this.extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).put(Constants.PREPARE_OPERATOR_ENV_KEY + namespaceInstallTo, true);
+            }
         }
     }
 
+    private void deleteClusterOperatorNamespace() {
+        LOGGER.info("Deleting of Namespace " + this.namespaceInstallTo);
```
Comment (Frawless) [inline systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java:400]: Use the same pattern as we use for other logs

**Review by Frawless — APPROVED** [review]

File: systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java (hunk 1 above)
Comment (see-quick) [reply systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java:400]: Like this?
```java
LOGGER.info("Deleting Namespace {}", this.namespaceInstallTo);
```

**Pull request merged**